index.rst

```diff
@@ -5,59 +5,108 @@ Technote Style and Writing Guide
 
-Technote Writing Guide
-----------------------
-Writing tempus
-^^^^^^^^^^^^^^
-**Suggestion:**
+*Example: Vibrations of the TMA structure immediately after a slew can impact the image quality performance, if they prolong into the time in which data is collected. We need to measure the residual vibration of the TMA in various axes, and in identical conditions as in real observations. These tests should be combined with those that measure the jittering of other mechanical and optical systems in the light path for an overall assessment of the 
+optical performance. The results in this technote should be regarded as preliminary because the final TMA settings have not been configured.*
 
-Write in present tense as much as possible.
+Related Tickets
+===============
 
-**Reason:**
+A bulleted list of related tickets and their titles, including those issued to  writing the technote itself, those related to relevant code used to produce the results and any other Operations tickets that can help trace the problem. 
 
-This gives the reader a more lively impression.
+*Example:*
 
-**Example:**
 
-???
+* `SITCOM-798 <https://jira.atlassian.net/browse/SITCOM-798>`_: *M1M3 - settling time after a slew*
+* `SITCOM-1172 <https://jira.atlassian.net/browse/SITCOM-1172>`_: *M1M3 - analyze settling times after a slew statistically*
 
+Related Requirements
+====================
 
-Writing perspective
-^^^^^^^^^^^^^^^^^^^
-**Suggestion:**
+(if applicable) A list of the requirements verified by the results of the technote, including code
+and title. The full description of the requirement could be added if it facilitates the
+interpretation of the results. 
 
-- Passive voice?
-- "We" perspective?
+*Example*:
 
-**Reason:**
+*LTS-88-REQ-0051: The positioning system SHALL be able to meet all its requirements within 3 seconds of ending a short slew (3.5 degrees in 2 seconds)*
 
-**Example:**
 
-Summary and Conclusion
+Execution Details and Data
+==========================
+
+Include scripts (with complete links to Github), data sets used (block number, source of data), and date in which they were executed (this is often helpful). Possibly a short description of procedure and/or how the script was executed, in case it requires a configuration file or specific arguments. The relevant test cases can be highlighted here as well. 
+
+Results and Discussion
 ======================
 
+This section can simply show numerical and graphical results, leaving the discussion for another section,  or incorporate also conclusions or take-aways from each of the items as they are presented.
+
+Specific analysis block 1
+-------------------------
+Use this kind of header and subheaders to organize the results and discussion.
+
```

Comment:
If there are more than 1, it would be useful to add what this specific section is looking at.

Reply:
Added a point to explain briefly the content of the section.